tools/fast_pusher_.py

```diff
@@ -61,6 +61,13 @@
 parser.add_argument('--oci', action='store_true',
                     help='Push the image with an OCI Manifest.')
 
+parser.add_argument('--certificates', nargs='*', help='A comma separated ' +
```

Comment:
This seems to disallow paths, which contain a comma, right? Any plans for an encoding?